apps/challenges/models.py

```diff
@@ -12,6 +14,11 @@
 class Challenge(TimeStampedModel):
 
     """Model representing a hosted Challenge"""
+
+    def __init__(self, *args, **kwargs):
+        super(Challenge, self).__init__(*args, **kwargs)
+        self.__original_evaluation_script = self.evaluation_script
```

Comment:
Is there any particular reason for using double underscore ? As far as I know, using a single underscore would have conveyed the meaning. What are your thoughts on this @deshraj ?